cmd/brew-test-bot.rb

```diff
@@ -1300,6 +1300,8 @@ def formulae
         end
       end
 
+      @all_formulae_unchanged_dependencies -= @formulae
```

Comment:
I think this logic would have to be reversed to have an effect on https://github.com/Homebrew/homebrew-core/pull/39225  (`-=` -> `+=`)

which would be equivalent to removing the `brew uninstall` lines at https://github.com/Homebrew/homebrew-test-bot/blob/master/cmd/brew-test-bot.rb#L1015-L1017

there may be a more surgical approach, but I haven't had time to figure out what it would be

Reply:
@scpeters Whoops, typo! Fixed that, mind retesting? Thanks again!